openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2293,6 +2268,178 @@ The following is a non-normative example of a Credential Response containing a C
 
 <{{examples/credential_response_sd_jwt_vc.txt}}
 
+# Claims Description 
+
+Claims description objects are used in two places in this specification, in the
+Credential Issuer metadata and in the Authorization Details. The following
+sections define the structure and semantics of claims description objects.
+
+## Claims Description for Authorization Details {#claims-description-authorization-details}
+
+A claims description object as used in authorization details is an object that defines the
+requirements for the claims that the Wallet requests to be included in the
+Credential.
+
+For a claims description object, the following keys defined by this
+specification can be used to describe the claim or claims (other keys MAY also
+be used):
+
+  * `path`: REQUIRED. Array. Claim path pointer as defined in (#claims_path_pointer)
+    to identify the claim(s) in the Credential.
+  * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the
+    Wallet will only accept a Credential that includes this claim. If set to
+    `false`, the claim is not required to be included in the Credential. If the
+    `mandatory` parameter is omitted, the default value is `false`.
+
+The rules defined in (##claims-description-processing) apply.
+
+## Claims Description for Issuer Metadata {#claims-description-issuer-metadata}
+
+A claims description object as used in the Credential Issuer metadata is an
+object used to describe how a certain claim in the Credential should be
+displayed to the End-User. It is used in the `claims`
+parameter in the Credential Issuer metadata defined in (#format-profiles). The
+following keys can be used to describe the claim or claims:
+
+  * `path`: REQUIRED if the Credential Format uses a JSON-based claims
+    structure; MUST NOT be present otherwise. The value MUST be a non-empty
+    array representing a claims path pointer that specifies the path to a claim
+    within the credential, as defined in (#claims_path_pointer).
+  * `namespace`: REQUIRED if the Credential Format is based on the mdoc format
+    defined in ISO 18013-5; MUST NOT be present otherwise. The value MUST be a
+    string that specifies the namespace of the data element within the mdoc,
+    e.g., `org.iso.18013.5.1`.
```

Comment:
```suggestion
  * `path`: REQUIRED if the Credential Format uses a JSON-based claims
    structure; MUST NOT be present otherwise. The value MUST be a non-empty
    array representing a claims path pointer that specifies the path to a claim
    within the credential, as defined in (#claims_path_pointer).
  * `namespace`: REQUIRED if the Credential Format is based on the mdoc format
    defined in ISO 18013-5; MUST NOT be present otherwise. The value MUST be a
    string that specifies the namespace of the data element within the mdoc,
    e.g., `org.iso.18013.5.1`.
```